google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreException.java

```diff
@@ -73,4 +78,10 @@ static FirestoreException networkException(IOException exception, boolean retrya
   static FirestoreException apiException(ApiException exception) {
     return new FirestoreException(exception);
   }
+
+  @VisibleForTesting
```

Comment:
Use `@InternalApi` (from GAX) instead of `@VisibleForTesting` (which is from Guava, which isn't supposed to be exposed on the surface of google-cloud-java)